Sources/TuistAnalytics/Backends/TuistAnalyticsCloudBackend.swift

```diff
@@ -29,7 +29,7 @@ class TuistAnalyticsCloudBackend: TuistAnalyticsBackend {
     }
 
     func send(commandEvent: CommandEvent) async throws {
-        guard config.options.contains(.analytics) else { return }
+        guard !config.options.contains(.disableAnalytics) else { return }
```

Comment:
You probably also want to edit the `TuistAnalyticsCloudBackendTests` 🙏

Reply:
Thank you. Updated.